charts/kargo/templates/controller/cluster-roles.yaml

```diff
@@ -50,7 +50,6 @@ rules:
   - list
   - patch
   - promote
-  - update
```

Comment:
The only update to Stages involves removing a finalizer and that can quite easily be done with a patch.